extensions/data-plane/data-plane-http/README.md

```diff
@@ -1,32 +1,51 @@
+| `edc.dataplane.http.sink.partition.size`  | Number of partitions for parallel message push in the `HttpDataSink` | false     | 5             |
 
-## Configuration properties
+## Design Principles
 
-* edc.dataplane.http.sink.partition.size - Sets the number o parallel partions for the sink (default set to 5).
+This extension provides implementations for `HttpDataSourceFactory`, `HttpDataSinkFactory`, `HttpDataSource` and `HttpDataSink`, which are triggered when
+the `DataFlowRequest` type is `HttpData`.
 
-## Data properties
+Basically the role of these classes is to extract the parameters (query params, path, body, auth header...) required to hit the HTTP endpoint and then to perform the call.
 
-see [HttpDataAddress.java](../../../spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/HttpDataAddress.java)
+The table below summarizes how each parameter is retrieved for the source/sink implementations.
 
-* type - The HTTP transfer type is "HttpData".
-* endpoint - The http endpoint.
-* name - The name associated with the HTTP data, typically a filename (optional).
-* authKey - The authentication key property name (optional).
-* authCode - The authentication code property name (optional).
-* secretName - The name of the vault secret that is containing the authorization code (optional).
-* proxyBody - If set to true the body of the actual request will be used to retrieve data from this address.
-* proxyPath - If set to true the path of the actual request will be used to retrieve data from this address.
-* proxyQueryParams - If set to true the query params of the actual request will be used to retrieve data from this address.
-* proxyMethod - If set to true the http method of the actual request will be used to retrieve data from this address.
-* header:* - The additional headers to use as json string e.g. ```"header:x-ms-blob-type": "BlockBlob"```. Note that Content-Type is handled separately - it will not be taken into consideration if specified here.
-* contentType - The HTTP Content-Type header is handled separate and has a default value of application/octet-stream.
\ No newline at end of file
+| Parameter                 |                                               `HttpDataSource`                                               | `HttpDataSink`                                                    | Example                              |
+|:--------------------------|:------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------|:-------------------------------------|
+| Base url                  |                                       Source/destination `DataAddress`                                       | Same                                                              | http://example.com                   |
```

Comment:
center alignment in the `HttpDataSource` column makes this really hard to read, could be left one used instead?